quai/quaiconfig/config.go

```diff
@@ -85,7 +85,7 @@ var Defaults = Config{
 		Recommit: 3 * time.Second,
 	},
 	TxPool:      core.DefaultTxPoolConfig,
-	RPCGasCap:   50000000,
+	RPCGasCap:   18000000, // Should be the same as the gas ceiling
```

Comment:
if it should be same as gas ceiling? cant you put the GasCeil here? instead of a value

Reply:
Done